sstables/sstable_directory.cc

```diff
@@ -61,6 +61,13 @@ sstable_directory::sstables_registry_components_lister::sstables_registry_compon
 {
 }
 
+sstable_directory::restore_components_lister::restore_components_lister(const data_dictionary::storage_options::value_type& options,
+                                                                        std::vector<sstring> toc_filenames)
+        : _prefix(std::get<data_dictionary::storage_options::s3>(options).prefix)
```

Comment:
It looks like this field is unused

Reply:
yeah, it's a leftover of a refactor. dropped.